website/django/data/views.py

```diff
@@ -98,28 +98,18 @@ def variant(request):
 def variant_reports(request, variant_id):
     variant_id = int(variant_id)
     query = Report.objects.filter(Variant_id=variant_id)
-
     '''
-    Uncomment the following code to filter enigma/bic submissions to clinvar/lovd
-
-    # filter out enigma submissions to clinvar/lovd and bic submissions to clinvar
-    filtered_query = []
+    NOTE: Uncomment to send report versions to UI
```

Comment:
we opted to not send reportdiffs to the UI for now (needs further review), hence the commented out code here.